src/libs/actions/Policy.js

```diff
@@ -22,8 +22,20 @@ Onyx.connect({
         if (!key) {
             return;
         }
-
         if (val === null || val === undefined) {
+            // If we are deleting a policy, we have to check every report linked to that policy
+            // and unset the draft indicator (pencil icon) alognside removing any draft comments
+            // More info: https://github.com/Expensify/App/issues/14260
```

Comment:
```suggestion
            // If we are deleting a policy, we have to check every report linked to that policy
            // and unset the draft indicator (pencil icon) alongside removing any draft comments. Clearing these values will keep the newly archived chats from being displayed in the LHN.
            // More info: https://github.com/Expensify/App/issues/14260
```
Good comment! I wanted to just add a little bit more here to explain _why_ we have to do those things.